external/header.php

```diff
@@ -34,21 +54,29 @@ public static function __callstatic($name, $arguments)
 // Only users from authorized IP addresses may control Profiling
 if ($controlIPs === false || in_array($_SERVER['REMOTE_ADDR'], $controlIPs) || PHP_SAPI == 'cli')
 {
-  if (isset($_GET['_profile']))
+  /* Backwards Compatibility getparam check*/
+  if( !isset($_xhprof['getparam']) )
+  {
+      $_xhprof['getparam'] = '_profile';
+  }
+  
+  if (isset($_GET[$_xhprof['getparam']]))
   {
     //Give them a cookie to hold status, and redirect back to the same page
-    setcookie('_profile', $_GET['_profile']);
-    $newURI = str_replace(array('_profile=1','_profile=0'), '', $_SERVER['REQUEST_URI']);
+    setcookie('_profile', $_GET[$_xhprof['getparam']]);
+    $newURI = str_replace(array($_xhprof['getparam'].'=1',$_xhprof['getparam'].'=0'), '', $_SERVER['REQUEST_URI']);
     header("Location: $newURI");
     exit;
   }
-
-  if (isset($_COOKIE['_profile']) && $_COOKIE['_profile'] || PHP_SAPI == 'cli' && ((isset($_SERVER['XHPROF_PROFILE']) && $_SERVER['XHPROF_PROFILE']) || (isset($_ENV['XHPROF_PROFILE']) && $_ENV['XHPROF_PROFILE'])))
+  
+  if (isset($_COOKIE['_profile']) && $_COOKIE['_profile'] || PHP_SAPI == 'cli' && ( (isset($_SERVER[$envVarName]) && $_SERVER[$envVarName]) 
```

Comment:
Why `||` in this line wasn't placed on it's own line as you did with other `||` later on?